diff --git a/customClient.go b/customClient.go
index f74153f..6f94f31 100644
--- a/customClient.go
+++ b/customClient.go
@@ -8,7 +8,9 @@ import (
 	"math/rand"
 	"strings"
 	"time"
-	//"go.etcd.io/etcd/clientv3"//this import is generating an error
+
+	"go.etcd.io/etcd/clientv3"
+	//this import is generating an error
 )
 
 type keyValue struct {
@@ -54,7 +56,7 @@ func (o *keyValue) createValue() {
 func createclient(putPercentage float64) {
 	//create client that does put/get at the ratio requested
 	// having the clientv3 used in anyway generates the same/similar error
-	//mt.Println(clientv3.Version())
+	fmt.Println(clientv3.Version("now"))
 }
 
 func (o *keyValue) applyCrc() {
